content/html/concepts/attributes/terms/title/title.md

```diff
@@ -0,0 +1,33 @@
+**Title**
```

Comment:
This entry is missing the Metadata section at the head of the file. To see what's required, review the [metadata section](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md#metadata) of the [content standards documentation](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md). Also look at our [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md) to see what we're looking for.